redhat-operators/businessautomation-operator/businessautomation-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,206 @@
+    spec:
+      permissions:
+        - serviceAccountName: business-automation-operator
+          rules:
+            - apiGroups:
+                - ""
+              resources:
+                - "*"
+              verbs:
+                - "*"
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+                - serviceaccounts
+              verbs:
+                - "*"
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - "*"
+            - apiGroups:
+                - app.kiegroup.org
+                - apps.openshift.io
+                - image.openshift.io
+                - build.openshift.io
+                - route.openshift.io
+              resources:
+                - "*"
+              verbs:
+                - "*"
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - roles
+                - rolebindings
+              verbs:
+                - "*"
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+      deployments:
+        - name: business-automation-operator
+          spec:
+            replicas: 1
+            strategy:
+              type: Recreate
+            selector:
+              matchLabels:
+                name: business-automation-operator
+            template:
+              metadata:
+                labels:
+                  name: business-automation-operator
+              spec:
+                serviceAccountName: business-automation-operator
+                containers:
+                  - name: business-automation-operator
+                    image: brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhpam-7-tech-preview/rhpam73-operator:1.0
+                    ports:
+                      - containerPort: 60000
+                        name: metrics
+                    command:
+                      - kie-cloud-operator
+                    imagePullPolicy: Always
+                    readinessProbe:
+                      exec:
+                        command:
+                          - stat
+                          - /tmp/operator-sdk-ready
+                      initialDelaySeconds: 4
+                      periodSeconds: 10
+                      failureThreshold: 1
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.namespace
```

Comment:
`fieldPath: metadata.annotations['olm.targetNamespaces']`

Reply:
This operator is only deployable in its own or single namespace. As such, i'd like to leave this as-is. Changing would also require additional code changes that aren't necessary for our operator to function at this point.